vm/vm.go

```diff
@@ -788,7 +856,14 @@ func operateStrings(op lex.Token, av, bv value.StringValue) value.Value {
 			return value.BoolValueFalse
 		}
 		return value.BoolValueTrue
-
+	case lex.TokenContains:
+		if len(a) == 0 || len(b) == 0 {
+			return value.BoolValueFalse
```

Comment:
Why do we special case empty strings? `"" contains ""` seems like a dumb way of implementing `foo == ""` where `foo` may be "" -- dumb, but valid?